plugins/player-activity-tracker.js

```diff
@@ -153,13 +159,16 @@ window.plugin.playerTracker.processNewData = function(data) {
         // portal the player is at, so ignore it.
         if(markup[1].plain.indexOf('destroyed the Link') !== -1
           || markup[1].plain.indexOf('destroyed a Control Field') !== -1
+          // COMM messages changed a bit, keep old rules ↑ in case of rollback
+          || markup[1].plain.indexOf('destroyed the') !== -1
```

Comment:
That won't ignore fields as they have "a" instead of "the" there
This will ignore both.
```suggestion
          || markup[1].plain.indexOf('destroyed ') !== -1
```

Haven't tested, not sure if it doesn't causes some problems 🤷

Reply:
`destroyed a resonator` is reliable

Reply:
and for field, it's also a `destroyed the`